build/bin/compile-protos.sh

```diff
@@ -2,7 +2,8 @@
 
 proto_imports=".:${GOPATH}/src/github.com/gogo/protobuf/protobuf:${GOPATH}/src/github.com/gogo/protobuf:${GOPATH}/src/github.com/google/protobuf/src:${GOPATH}/src:${GOPATH}/src/github.com/OpenNetworkingFoundation/gmap-ran/"
 
-protoc -I=$proto_imports --gogofaster_out=Mgoogle/protobuf/duration.proto=github.com/gogo/protobuf/types,import_path=ran/e2,plugins=grpc:. api/e2/*.proto
+rm -f api/e2/e2-interface.proto api/e2/e2-interface.pb.go
+wget https://raw.githubusercontent.com/onosproject/onos-ran/master/api/sb/e2-interface.pb.go -P api/e2
```

Comment:
maybe you should pull the proto file, and compile it here like the others? And it might be ok just to file copy it, as this would allow local changes without having to check it in first